pyuoi/linear_model/poisson.py

```diff
@@ -494,3 +494,20 @@ def adjusted_response(X, y, coef, intercept=0):
         w = np.exp(Xbeta)
         z = Xbeta + (y / w) - 1
         return w, z
+
+    def _fit_intercept_no_features(self, y):
+        """"Fit a model with only an intercept.
+
+        This is used in cases where the model has no support selected.
+        """
+        return PoissonInterceptFitter(y)
+
+class LogisticInterceptFitter(object):
+    def __init__(self, y):
+        raise NotImplementedError
```

Comment:
@pssachdeva Do you have the poisson expression handy?

Reply:
for the intercept with no feature input

Reply:
mu = np.exp(np.dot(X, self.coef_))
self.intercept_ = np.log(np.mean(y)/np.mean(mu))

with features... if there's no features, i believe it should just be the log of the mean response variable.